src/vs/editor/browser/services/hoverService/hover.css

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.monaco-workbench .workbench-hover {
+.monaco-hover.workbench-hover {
```

Comment:
Why do you sometimes not prefix with `.monaco-hover` in this file, isn't it all contained within?